examples/prometheus/README.md

```diff
@@ -0,0 +1,55 @@
+# Integrating the power limit optimizer with ImageNet training
+
+This example will demonstrate how to integrate Zeus with `torchvision` and the ImageNet dataset.
+
+[`train_single.py`](train_single.py) and [`train_dp.py`](train_dp.py) were adapted and simplified from [PyTorch's example training code for ImageNet](https://github.com/pytorch/examples/blob/main/imagenet/main.py).
+The former script is for simple single GPU training, whereas the latter is for data parallel training with PyTorch DDP and [`torchrun`](https://pytorch.org/docs/stable/elastic/run.html).
+
+## Dependencies
+
+All packages (including torchvision and prometheus_client) are pre-installed if you're using our [Docker image](https://ml.energy/zeus/getting_started/environment/).
+You just need to download and extract the ImageNet data and mount it to the Docker container with the `-v` option (first step below).
+
+1. Download the ILSVRC2012 dataset from [the ImageNet homepage](http://www.image-net.org/).
+    Then, extract archives using [this script](https://github.com/pytorch/examples/blob/main/imagenet/extract_ILSVRC.sh) provided by PyTorch.
+1. Install `zeus` and build the power monitor, following [Installing and Building](https://ml.energy/zeus/getting_started/installing_and_building/).
+1. Install `torchvision`:
+    ```sh
+    pip install torchvision==0.15.2
+    ```
+1. Install `prometheus_client`:
+    ```sh
+    pip install prometheus-client
+    ```
+
+## EnergyHistogram, PowerGauge, and EnergyCumulativeCounter
+- [`EnergyHistogram`][zeus.metric.EnergyHistogram]: Records energy consumption data for GPUs, CPUs, and DRAM and pushes the data to Prometheus as histogram metrics. This is useful for tracking energy usage distribution over time.
```

Comment:
If you check how Markdown is rendered on GitHub ([here](https://github.com/ml-energy/zeus/blob/f032488d9c1d98375db44fa7a919eda79a064a00/examples/prometheus/README.md)), you'll see that this kind of link is not rendered correctly. The bracket syntax is only for Mkdocs. GitHub isn't smart enough to automatically understand what `zeus.metric.EnergyHistogram` is and link to the related reference doc. Instead, you should manually link the corresponding API documentation link:

```markdown
[`ZeusMonitor`](https://ml.energy/zeus/reference/monitor/energy/#zeus.monitor.energy.ZeusMonitor)
```

Since `zeus.metrics` has not landed in `master` yet, there is no corresponding live API reference documentation under https://ml.energy/zeus. Thus you should open up `scripts/preview_docs.sh` and construct the right URL for the three classes.